tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
+            # poplate the volume options
+            reg_ex = re.compile("^volume[0-9]+.options+")
```

Comment:
Well, my thought was to process and update the volume options along with other volume attributes and save additional updates to etcd. If you are convinced that this is the right way to do this, I am fine with it